src/pages/home/sidebar/SidebarBottom.js

```diff
@@ -57,7 +58,13 @@ const SidebarBottom = ({myPersonalDetails, network, insets}) => {
                     source={{uri: myPersonalDetails.avatarURL}}
                     style={[styles.actionAvatar]}
                 />
-                <View style={StyleSheet.flatten(indicatorStyles)} />
+                {
+                    Platform.OS === 'web' ? (
```

Comment:
The use of `Platform` here works but is not how we do this in our codebase. To create a cross platform component we must create two files one for web (`index.js`) and one for native (`index.native.js`). Please review this README section -> https://github.com/Expensify/Expensify.cash#platform-specific-file-extensions